corehq/apps/app_manager/models.py

```diff
@@ -4467,6 +4514,14 @@ def get_hierarchy(case_type):
 
         return meta
 
+    def get_subcase_types_of_module(self, module):
```

Comment:
Could make this more generic by just passing the case type rather than the module.